### PR TITLE
In progress improvements to definitions

### PR DESCRIPTION
**Pull request by timlinux**
This fixes missing classifications for some hazards and deals with hazard classifications that use dicts rather than discrete values...

screen shot 2017-01-25 at 5 30 18 pm

screen shot 2017-01-25 at 5 29 55 pm

### CODE REVIEW

File: safe/gui/tools/help/definitions_help.py [hunk 1]
```diff
@@ -531,47 +554,63 @@ def definition_to_message(definition, heading_style=None):
...
 
     if 'multi_event_notes' in definition:
-        message.add(m.Paragraph(
-            m.ImportantText(tr('Notes for multi events / scenarios:'))))
-        bullets = m.BulletedList()
-        for note in definition['multi_event_notes']:
-            bullets.add(m.Text(note))
-        message.add(bullets)
+        message.add(
+            m.Heading(tr('Notes for multi events / scenarios:'),
```
Comment (Gustry) [inline safe/gui/tools/help/definitions_help.py]: I think you should pull left

**Comment by timlinux** [comment]
Thanks @Gustry fixed. Also added class colours to the tables

2017-01-25 23_29_29-inasafe 4 0 0b0 help

**Comment by Charlotte-Morgan** [comment]
very nice - this is growing on me and now I'm a fan :)

**Comment by Gustry** [comment]
@timlinux I think you should display the not exposed class too : https://github.com/inasafe/inasafe/blob/develop/safe/definitions/hazard_classifications.py#L32

**Comment by timlinux** [comment]
> very nice - this is growing on me and now I'm a fan :)

@Charlotte-Morgan yay :-)

>  I think you should display the not exposed class too : 

@Gustry When in the comments it says 'automatically added at run time' ...... why don't we just add it to each definition?

**Comment by timlinux** [comment]
Ok updated to include the not exposed class now too:

2017-01-26 13_13_31-qgis 2 14 11-essen

File: safe/definitions/hazard_classifications.py [hunk 2]
```diff
@@ -29,17 +29,23 @@
 __revision__ = '$Format:%H$'
 
 # This class will be automatically added to a hazard classification on runtime.
+# We do not include it in the classes below because we do not want the user
+# to be presented with not exposed in the keywords when setting up
+# their classes.
 not_exposed_class = {
     'key': 'not exposed',
     'name': tr('Not exposed'),
     'description': tr('Not exposed'),
```
Comment (Gustry) [inline safe/definitions/hazard_classifications.py]: Feel free to add some description ;-)

**Comment by Gustry** [comment]
Great, thanks

**Pull request merged**